src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -5,7 +5,7 @@
 
 #include "node/include/addon.hpp"
 #include "node/include/errors.hpp"
-#include "node/include/node_output.hpp"
+#include "node/include/helper.hpp"
```

Comment:
Shouldn't be deleted
```suggestion
#include "node/include/node_output.hpp"
#include "node/include/helper.hpp"
```

Reply:
@vishniakov-nikolai  This needed Formatting,So I formatted.